freeglut/freeglut/CMakeLists.txt

```diff
@@ -391,12 +396,22 @@ ENDIF()
 
 SET(LIBNAME freeglut)
 IF(WIN32)
+    IF(FREEGLUT_REPLACE_GLUT)
+        SET(LIBNAME glut)
+    ELSE()
+        SET(LIBNAME freeglut)
+    ENDIF()
+
     LIST(APPEND LIBS winmm)
     IF(FREEGLUT_BUILD_SHARED_LIBS)
         TARGET_COMPILE_DEFINITIONS(freeglut PRIVATE FREEGLUT_EXPORTS)
+        SET_TARGET_PROPERTIES(freeglut PROPERTIES OUTPUT_NAME ${LIBNAME})
     ENDIF()
     IF(FREEGLUT_BUILD_STATIC_LIBS)
         TARGET_COMPILE_DEFINITIONS(freeglut_static PUBLIC FREEGLUT_STATIC)
+        IF(FREEGLUT_REPLACE_GLUT AND NOT FREEGLUT_BUILD_SHARED_LIBS)
+            SET_TARGET_PROPERTIES(freeglut_static PROPERTIES OUTPUT_NAME ${LIBNAME})
```

Comment:
As far as I remember MinGW in some cases generates both *.dll  and *.a files. And in this case *.a file is not a static library but additional file for proper linking with shared library. So I decided to add this check to avoid file name collisions. But I have not tested build of both (shared+static) libraries simultaneously, so I am not sure if this check is really necessary. I'll test it now.